### PR TITLE
v4.0.0-alpha.1

### PR DESCRIPTION
**Pull request by dtassone**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,14 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+# [4.0.0-alpha.1](https://github.com/dtassone/material-ui-x/compare/v0.1.67...v4.0.0-alpha.1) (2020-09-17)
+
+**Note:** Version bump only for package material-ui-x
```
Comment (oliviertassinari) [inline CHANGELOG.md:8]: So far, I have used this script to generate the changelog

https://trello.com/c/uspnIWkh/1566-release-note

Running on this page: https://github.com/mui-org/material-ui-x/compare/v0.1.67...master

**Review by oliviertassinari — COMMENTED** [review]
Netlify env has memory issues, it seems that we can ignore the fail.

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,14 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+# [4.0.0-alpha.1](https://github.com/dtassone/material-ui-x/compare/v0.1.67...v4.0.0-alpha.1) (2020-09-17)
```
Comment (oliviertassinari) [inline CHANGELOG.md:6]: Wrong link

File: CHANGELOG.md (hunk 1 above)
Comment (dtassone) [reply CHANGELOG.md]: What does this script do? How do you run it?

Comment (oliviertassinari) [reply CHANGELOG.md]: It's one way to convert the git history into a changelog:

Capture d’écran 2020-09-17 à 15 01 54

Comment (dtassone) [reply CHANGELOG.md]: image

Comment (oliviertassinari) [reply CHANGELOG.md]: Wrong page:

Capture d’écran 2020-09-17 à 15 21 02

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,438 +3,302 @@
...
+Fix issue #254, focus cell fully visible (#256)  
+[DataGrid] fix issues with path and import (#259)  
+[DataGrid] Replace style-components with @material-ui/styles (#168)  
+[docs] Migrate Getting Started section (#255)  
+[docs] Migrate Selection pages (#248)  
+[docs] Migrate more pages (#243)  
+[Security] Bump bl from 4.0.2 to 4.0.3 (#241)  
+fix build  
+[docs] Apply review from Matt  
+hack around
```
Comment (oliviertassinari) [inline CHANGELOG.md:35]:
```suggestion
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -3,438 +3,302 @@
...
+[DataGrid] Move column resizing to XGrid only (#257)  
+[docs] Polish the first experience (#261)  
+Fix issue #254, focus cell fully visible (#256)  
+[DataGrid] fix issues with path and import (#259)  
+[DataGrid] Replace style-components with @material-ui/styles (#168)  
+[docs] Migrate Getting Started section (#255)  
+[docs] Migrate Selection pages (#248)  
+[docs] Migrate more pages (#243)  
+[Security] Bump bl from 4.0.2 to 4.0.3 (#241)  
+fix build
```
Comment (oliviertassinari) [inline CHANGELOG.md:33]:
```suggestion
```

**Review by oliviertassinari — COMMENTED** [review]
I think that it would be great to group the changes, moving [docs] and [core] to the bottom as less relevant to developers. I also think that the changes should be sorted ASC (handled by the script) to make related changes together. 

A baseline example: https://github.com/mui-org/material-ui/releases/tag/v5.0.0-alpha.9

**Review by oliviertassinari — APPROVED** [review]

File: CHANGELOG.md [hunk 5]
```diff
@@ -3,438 +3,40 @@
...
 
-## [0.1.67](https://github.com/oliviertassinari/material-ui-x/compare/v0.1.66...v0.1.67) (2020-08-24)
+# [4.0.0-alpha.1](https://github.com/mui-org/material-ui-x/compare/v0.1.67...v4.0.0-alpha.1) (2020-09-17)
+
+## @material-ui/x-grid@v4.0.0-alpha.1 / @material-ui/data-grid@v4.0.0-alpha.1
+
+- [DataGrid] Add api pages for data-grid and x-grid (#289) @dtassone
+- [DataGrid] Add dark mode scrollbar (#282) @dtassone
+- [DataGrid] Better explain the limits of MIT vs commercial (#225) @oliviertassinari
+- [DataGrid] first v4 alpha version (#291) @dtassone
```
Comment (oliviertassinari) [inline CHANGELOG.md:13]:
```suggestion
- [DataGrid] First v4 alpha version (#291) @dtassone
```

**Pull request merged**